agent/tool-scripts/pcp

```diff
@@ -7,11 +7,16 @@ import os
 PROG = os.path.basename(sys.argv[0])
 
 if len(sys.argv) != 2 or sys.argv[1] != "--help":
-    print(f"{PROG}: This script is deprecated, please run it with --help for info on registering the tool.", file=sys.stderr)
-    print(f"{PROG}: Run /opt/pbench-agent/tool-scripts/pcp --help for more info.", file=sys.stderr)
+    print(
+        f"{PROG}: This script is deprecated, please run it with --help for info on registering the tool.",
+        file=sys.stderr,
+    )
+    print(
+        f"{PROG}: Run /opt/pbench-agent/tool-scripts/pcp --help for more info.",
+        file=sys.stderr,
+    )
```

Comment:
These look redundant...I think we would be better off with only one of them (probably the first).

Reply:
And it is not deprecated, this is just a place-holder to fit the new "pcp" into the legacy framework.

Reply:
I'd rather address this is in future to refactor all this stuff.